gems/aws-sdk-core/spec/aws/util_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+module Aws
+  describe Util do
```

Comment:
We should probably add tests for the other stuff while we're at it..

Reply:
Yeah, I think I plan to update this in a separate PR since we want this fix out with today's release.